docs/libs/ml/quickstart.md

```diff
@@ -24,4 +24,198 @@ under the License.
+member which represents the label, which could be the class in a classification problem, or the dependent
+variable for a regression problem.
+
+# TODO: Get dataset that has separate train and test sets
+As an example, we can use the Breast Cancer Wisconsin (Diagnostic) Data Set, which you can
+[download from the UCI ML repository](http://archive.ics.uci.edu/ml/machine-learning-databases/breast-cancer-wisconsin/breast-cancer-wisconsin.data).
+
+We can load the data as a `DataSet[String]` first:
+
+{% highlight scala %}
+
+val cancer = env.readCsvFile[(String, String, String, String, String, String, String, String, String, String, String)]("/path/to/breast-cancer-wisconsin.data")
+
+{% endhighlight %}
+
+The dataset has some missing values indicated by `?`. We can filter those rows out and
+then transform the data into a `DataSet[LabeledVector]`. This will allow us to use the
+dataset with the FlinkML classification algorithms.
+
+{% highlight scala %}
+
+val cancerLV = cancer
+  .map(_.productIterator.toList)
+  .filter(!_.contains("?"))
+  .map{list =>
+    val numList = list.map(_.asInstanceOf[String].toDouble)
+    LabeledVector(numList(11), DenseVector(numList.take(10).toArray))
+    }
+
+{% endhighlight %}
+
+We can then use this data to train a learner.
+
+A common format for ML datasets is the LibSVM format and a number of datasets using that format can be
+found [in the LibSVM datasets website](http://www.csie.ntu.edu.tw/~cjlin/libsvmtools/datasets/). FlinkML provides utilities for loading
+datasets using the LibSVM format through the `readLibSVM` function available through the MLUtils object.
+You can also save datasets in the LibSVM format using the `writeLibSVM` function.
+Let's import the Adult (a9a) dataset. You can download the 
+[training set here](http://www.csie.ntu.edu.tw/~cjlin/libsvmtools/datasets/binary/a8a)
+and the [test set here](http://www.csie.ntu.edu.tw/~cjlin/libsvmtools/datasets/binary/a8a.t).
+
+We can simply import the dataset then using:
+
+{% highlight scala %}
+
+val adultTrain = MLUtils.readLibSVM("path/to/a8a")
+val adultTest = MLUtils.readLibSVM("path/to/a8a.t")
+
+{% endhighlight %}
+
+This gives us a `DataSet[LabeledVector]` that we will use in the following section to create a classifier.
+
+Due to an error in the test dataset we have to adjust the test data using the following code, to 
+ensure that the dimensionality of all test examples is 123, as with the training set:
+
+{% highlight scala %}
+
+val adjustedTest = adultTest.map{lv =>
+      val vec = lv.vector.asBreeze
+      val padded = vec.padTo(123, 0.0).toDenseVector
+      val fvec = padded.fromBreeze
+      LabeledVector(lv.label, fvec)
+    }
+
+{% endhighlight %}
+
+## Classification
+
+Once we have imported the dataset we can train a `Predictor` such as a linear SVM classifier.
+
+{% highlight scala %}
+
+val svm = SVM()
+  .setBlocks(env.getParallelism)
+  .setIterations(100)
+  .setRegularization(0.001)
+  .setStepsize(0.1)
+  .setSeed(42)
```

Comment:
Maybe we should also add again how to setup a project using FlinkML. E.g. saying which dependencies users have to add to their `pom.xml`.